pandas/core/base.py

```diff
@@ -990,11 +990,17 @@ def factorize(self, sort=False, na_sentinel=-1):
         from pandas.core.algorithms import factorize
         return factorize(self, sort=sort, na_sentinel=na_sentinel)
 
-    def searchsorted(self, key, side='left'):
-        """ np.ndarray searchsorted compat """
+    def searchsorted(self, key, side='left', sorter=None):
```

Comment:
1) Does `Series` not do something explicit with `sorter` in its implementation?
2) Unfortunately, I don't believe the doc strings are inherited at the moment (i.e. on `master`) because if you type `help(DatetimeIndex.searchsorted)` and `help(IndexOpsMixin.searchsorted)`, the former returns an string, while the other returns `np.ndarray searchsorted compat`.